packages/react-native/Libraries/AppDelegate/RCTRootViewFactory.mm

```diff
@@ -241,11 +241,52 @@ - (void)didCreateContextContainer:(std::shared_ptr<facebook::react::ContextConta
   contextContainer->insert("ReactNativeConfig", _reactNativeConfig);
 }
 
+- (NSArray<id<RCTBridgeModule>> *)extraModulesForBridge:(RCTBridge *)bridge {
+  if (_configuration.extraModulesForBridge != nil) {
+    return _configuration.extraModulesForBridge(bridge);
+  }
+  return nil;
+}
+
+- (NSDictionary<NSString *,Class> *)extraLazyModuleClassesForBridge:(RCTBridge *)bridge {
+  if (_configuration.extraLazyModuleClassesForBridge != nil) {
+    return _configuration.extraLazyModuleClassesForBridge(bridge);
+  }
+  return nil;
+}
+
 - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 {
+  if (_configuration.sourceURLForBridge != nil) {
+    return _configuration.sourceURLForBridge(bridge);
+  }
   return [self bundleURL];
 }
 
+- (BOOL)bridge:(RCTBridge *)bridge didNotFindModule:(NSString *)moduleName
```

Comment:
i think it'd be nice to pragma mark which of these methods are for RCTBridgeDelegate